src/types.ts

```diff
@@ -129,7 +129,7 @@ export interface BuildContext {
 /** Documentation: https://esbuild.github.io/api/#serve-return-values */
 interface ServeResult {
   port: number;
-  host: string;
+  hosts: string[];
```

Comment:
This seems to have changed in the latest `esbuild`, [the docs say](https://esbuild.github.io/api/#serve-return-values) it should be a string array called `hosts` now.